### PR TITLE
Release/1.1.4

### PR DESCRIPTION
**Pull request by QuintinWillison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,13 +268,18 @@ As the library is actively developed couple of features are not there yet:
 
 This library uses [semantic versioning](http://semver.org/). For each release, the following needs to be done:
 
-* Create a branch for the release, named like `release-1.1.4`
+* Create a branch for the release, named like `release/1.1.5`
```
Comment (gernest) [inline README.md:271]: why `release/1.1.5` instead of `release-1.1.5` ? I mean any reason to use `/` instead of `-`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
 # Change Log
 
+## [v1.1.4](https://github.com/ably/ably-go/tree/v1.1.4) (2020-04-24)
+
+[Full Changelog](https://github.com/ably/ably-go/compare/v1.1.3...v1.1.4)
+
+**Implemented enhancements:**
+
+- Implement host fallback [\#53](https://github.com/ably/ably-go/issues/53)
```
Comment (tcard) [inline CHANGELOG.md:9]: I just closed the issue, but this was done at #78 two years ago.

File: README.md (hunk 1 above)
Comment (QuintinWillison) [reply README.md:271]: We had an internal discussion on this a few months back and have decided to start using forward slashes in branch names to delimit categories of branch like this. At the time we updated our internal document [Development flow for Ably repos](https://wiki.ably.io/pages/Development-flow-for-Ably-repos) and I will share that as a snapshot with you via Slack now. The task of formally documenting this in the public domain is [in my backlog](https://github.com/ably/docs/issues/577).

File: CHANGELOG.md (hunk 2 above)
Comment (QuintinWillison) [reply CHANGELOG.md:9]: Well spotted. Thanks! I don't want to start retroactively trying to modify the old change history but equally removing this line from this release PR may end up confusing the change log generator for future releases (actually, perhaps not using the delta option that I prefer using these days!). So, yes, I'll just remove the line as it's not helpful to those looking or a 'human readable' change log that tells them what's new! `</ monologue>`

Comment (QuintinWillison) [reply CHANGELOG.md]: https://github.com/ably/ably-go/pull/158/commits/694511d0905b06c1af23f6026008268641a9eba0

**Comment by QuintinWillison** [comment]
@tcard and @gernest - any chance you can resolve conversations if you're happy with my actions and then approve if you also happy to do so? Thanks.

**Pull request merged**